src/main/java/net/cytonic/cytosis/events/ServerEventListeners.java

```diff
@@ -23,10 +24,15 @@ public static void initServerEvents() {
         Cytosis.getEventHandler().registerGlobalEvent(PlayerSpawnEvent.class, event -> {
             final Player player = event.getPlayer();
             if (CytosisSettings.LOG_PLAYER_IPS)
-                Logger.info(event.getPlayer().getUsername() + " (" + event.getPlayer().getUuid() + ") joined with the ip: " + player.getPlayerConnection().getServerAddress());
+                Logger.info(STR."\{event.getPlayer().getUsername()} (\{event.getPlayer().getUuid()}) joined with the ip: \{player.getPlayerConnection().getServerAddress()}");
             else
-                Logger.info(event.getPlayer().getUsername() + " (" + event.getPlayer().getUuid() + ") joined.");
+                Logger.info(STR."\{event.getPlayer().getUsername()} (\{event.getPlayer().getUuid()}) joined.");
             player.sendMessage("Hello!");
         });
+        Logger.info("Registering player chat event.");
+        Cytosis.getEventHandler().registerGlobalEvent(PlayerChatEvent.class, event -> {
+            final Player player = event.getPlayer();
+            Cytosis.getManager().getDatabase().addChat(player.getUuid(),event.getMessage());
+        });
```

Comment:
I can't help but notice that you do not in fact check if chat logging is enabled in the config.
'CytosisSettings.LOG_PLAYER_CHAT`